tasks/diff.py

```diff
@@ -138,14 +159,30 @@ def go_deps(ctx, baseline_ref=None, report_file=None):
             ]
             for binary, details in BINARIES.items():
                 for combo in details["platforms"]:
+                    flavor = details.get("flavor", AgentFlavor.base)
+                    build = details.get("build", binary)
                     platform, arch = combo.split("/")
                     goos, goarch = GOOS_MAPPING.get(platform), GOARCH_MAPPING.get(arch)
                     target = f"{binary}-{goos}-{goarch}"
                     prettytarget = f"{binary} {goos}/{goarch}"
 
+                    tags = [
+                        f"build:{build}",
+                        f"flavor:{flavor.name}",
+                        f"os:{goos}",
+                        f"arch:{goarch}",
+                        f"git_ref:#{commit_sha}",
```

Comment:
In the metric defined in `go_deps.py` we use `f"git_sha:{git_sha}",`, we might as well keep the same tag name (`git_sha`) and remove the `#` to be coherent ?

Reply:
Nice catch 👍